csfieldguide/static/scss/translation-rotation-interactives/translation-rotation.scss

```diff
@@ -0,0 +1,152 @@
+img {
+    width: 30px;
+}
+
+
+.interactive-body {
+    font-family: "Roboto",sans-serif;
+    font-size: 17px;
+    background-color: #f0f0f0;
+    margin: 0px;
+    overflow: hidden;
+    text-align: center;
+}
+
+label {
+    display: block;
+    font-size: 20px;
+    margin: 10px;
+}
+
+
+.button-manip {
+    color: #fff;
+    background-color: #ff7043;
+    border: none;
+    border-radius: 2px;
+    box-shadow: 0 2px 5px 0 rgba(0,0,0,0.16);
+    margin: 5px;
+    padding: 5px;
+}
+
+.little-button {
+    width: 32px;
+    height: 32px;
+    margin: 0px;
+    margin-bottom: 1px;
+}
+
+label, #mob-x, #mob-y, #mob-z {
+    font-family: "Cambria",Georgia,serif;
+    font-style: italic;
+    font-size: 15px;
+}
+
+
+#restart-button {
+    display: none;
+    margin: 10px auto;
+    font-size: 20px;
+}
+
+
+#user-input {
+    color: #fff;
+    background-color: #3f51b5;
+    padding: 5px;
+}
+
+
+#coordinates {
+    padding-right: 5px;
+}
+
+
+#coordinates input {
+    width: 3rem;
+    display: inline-block;
+    border: none;
+    border-radius: 2px;
+    box-shadow: 0 2px 5px 0 rgba(0,0,0,0.16);
+    height: 30px;
+    padding: 2px;
+}
+
+
+#mobile-coord {
+    display: none;
+}
+
+
+#desktop-coord {
+    display: none;
+}
+
+
+#mob-x, #mob-y, #mob-z {
+    display: inline-block;
+    font-size: 15px;
+    margin: 2px;
+}
+
+
+#symbol-grid, .row {
+    display: block;
+}
+
+
+#code-template img {
+    display: inline;
+    background-color: #fff;
+}
+
+
+#code-template button {
+    display: inline-block;
+}
+
+
+#end-buttons {
+    display: block;
```

Comment:
Same here with d-block class, you get the idea! Check other classes to see what you can replace :)